### ISSUE TITLE
vk.xml contains definition for StdVideoEncodeH264RefMgmtFlags which doesn't exist

### ISSUE THREAD

**Issue by NogginBops** [KhronosGroup/Vulkan-Docs#2388]
On line 7125 of vk.xml
https://github.com/KhronosGroup/Vulkan-Docs/blob/main/xml/vk.xml#L7125 
The following like appears:
```xml
        <type requires="vk_video/vulkan_video_codec_h264std_encode.h" name="StdVideoEncodeH264RefMgmtFlags"/>
```

The type defined (`StdVideoEncodeH264RefMgmtFlags`) seems like it's removed from the H264 video extension or might have never existed?

I'm quite confused why it is part of the spec. 

Should this line be removed? or is there something I'm missing?

**Comment by NogginBops** [comment]
It also seems like `vk.xml` forgets to include `StdVideoEncodeH264ReferenceListsInfoFlags`?

**Comment by NogginBops** [comment]
`StdVideoEncodeH265ReferenceModificationFlags` is also referenced but doesn't exist?

**Comment by oddhack** [comment]
At least some of those are dangling reference from when the APIs changed in the provisional period. @aqnuep, can you handle this since you know exactly what those changes were?

**Comment by aqnuep** [comment]
Sure. Thanks, @oddhack, for pinging me.

**Comment by oddhack** [comment]
This should be fixed in the 1.3.291 spec update.
